### PR TITLE
Use a custom action

### PR DESCRIPTION
**Pull request by chadwhitacre**
Easier to iterate this way

### CODE REVIEW

File: .github/workflows/enforce-license-compliance.yml [hunk 1]
```diff
@@ -8,31 +8,7 @@ on:
...
-          # to set it in one place in case we ever do need to roll it. 🤷
-
-          curl -O https://raw.githubusercontent.com/getsentry/.github/main/.github/workflows/FOSSA_API_KEY
-          echo "::set-output name=key::$(cat FOSSA_API_KEY | grep -v '#')"
-
-      - name: 'Checkout Code'
-        uses: actions/checkout@v2
-
-      - name: 'Run FOSSA Scan'
-        uses: fossas/fossa-action@v1.1.0
```
Comment (billyvg) [inline .github/workflows/enforce-license-compliance.yml:30]: You should pin these to a commit hash

**Review by kamilogorek — APPROVED** [review]

**Review by billyvg — APPROVED** [review]

File: .github/workflows/enforce-license-compliance.yml (hunk 1 above)
Comment (kamilogorek) [reply .github/workflows/enforce-license-compliance.yml:30]: Then it's not making it any easier to iterate tbh :P

Comment (chadwhitacre) [reply .github/workflows/enforce-license-compliance.yml:30]: Will do that over in the action.

Comment (chadwhitacre) [reply .github/workflows/enforce-license-compliance.yml:30]: Done in https://github.com/getsentry/action-enforce-license-compliance/commit/b4fde989274b441d2b747d9b19067fe329bdf432.

> Then it's not making it any easier to iterate

This is referring to the upstream `fossas/fossa-action` that we don't control. References to `getsentry/action-enforce-license-compliance` can use a movable pointer such as `main` on this PR (I'm thinking for other repos to point to a `latest` branch/tag).

**Pull request merged**